integrations/cohere/src/haystack_integrations/components/embedders/cohere/document_embedder.py

```diff
@@ -8,7 +8,7 @@
 from haystack.utils import Secret, deserialize_secrets_inplace
 from haystack_integrations.components.embedders.cohere.utils import get_async_response, get_response
 
-from cohere import COHERE_API_URL, AsyncClient, Client
+from cohere import AsyncClient, Client
```

Comment:
`COHERE_API_URL` is no longer exported from the SDK. The Cohere team recommends using a string literal in the code to replace it.

Reply:
I checked the examples we have in our documentation pages and none of them sets `COHERE_API_URL` so there are no changes required in the documentation pages.